pkg/registry/core/pod/storage/eviction_test.go

```diff
@@ -155,41 +146,6 @@ func TestEviction(t *testing.T) {
 	}
 }
 
-type FailDeleteUpdateStorage struct {
-	storage.Interface
-}
-
-func (f FailDeleteUpdateStorage) Delete(ctx context.Context, key string, out runtime.Object, precondition *storage.Preconditions, validateDeletion storage.ValidateObjectFunc) error {
-	return storage.NewKeyNotFoundError(key, 0)
-}
-
-func (f FailDeleteUpdateStorage) GuaranteedUpdate(ctx context.Context, key string, ptrToType runtime.Object, ignoreNotFound bool,
-	preconditions *storage.Preconditions, tryUpdate storage.UpdateFunc, suggestion ...runtime.Object) error {
-	return storage.NewKeyNotFoundError(key, 0)
-}
-
-var scheme = runtime.NewScheme()
-var codecs = serializer.NewCodecFactory(scheme)
-
-func newFailDeleteUpdateStorage(t *testing.T) (*REST, *etcd3testing.EtcdTestServer) {
```

Comment:
This code existed for a good reason, I find it very dubious that it can be removed like this ... Looking into it.